### PR TITLE
Balance the skills for programming

### PR DESCRIPTION
**Pull request by hoagrawal**
Added few lines to balance the programming skill in Job Spec

### CODE REVIEW

File: Recruitment/JobSpecs/2_why_you_platform_engineer.md [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
-* Ansible
-* Terraform
-* Java
+* Bluemix, CloudFoundry
+* Docker
+* Ansible, Terraform
 * Jenkins
 * Splunk
-* New Relic
+* New Relic, AppDynamics
```
Comment (aochsner) [inline Recruitment/JobSpecs/2_why_you_platform_engineer.md]: want to just lump Splunk w/ these as well?

File: Recruitment/JobSpecs/2_why_you_platform_engineer.md [hunk 2]
```diff
@@ -5,22 +5,22 @@
...
+* Cloud, \*aaS
 * CI/CD
-* \*aaS
 * Monitoring
 * Analytics
-* Automation
+* Comfortable with technical implementation to high-level architecture.
 
 ### We currently use, but aren’t wedded to:
+* Java, Spring MVC, SpringBoot
```
Comment (aochsner) [inline Recruitment/JobSpecs/2_why_you_platform_engineer.md]: Maybe add Node.js?  We've been doing that on the server side w/ our products & accelerators

**Comment by aochsner** [comment]
LGTM... added a couple suggestive comments

File: Recruitment/JobSpecs/2_why_you_platform_engineer.md (hunk 2 above)
Comment (hoagrawal) [reply Recruitment/JobSpecs/2_why_you_platform_engineer.md]: Good point. will add that

File: Recruitment/JobSpecs/2_why_you_platform_engineer.md (hunk 1 above)
Comment (hoagrawal) [reply Recruitment/JobSpecs/2_why_you_platform_engineer.md]: yup, sure.

**Pull request merged**